keel-slack/src/main/kotlin/com/netflix/spinnaker/keel/slack/handlers/GitDataGenerator.kt

```diff
@@ -36,6 +41,25 @@ class GitDataGenerator(
   fun generateConfigUrl(application: String): String =
     "${config.baseApiUrl}/managed/application/$application/config"
 
+  /**
+   * Builds a modal with the full commit message
+   */
+  fun buildFullCommitModal(message: String, hash: String): View {
+    return view { thisView -> thisView
+      .callbackId("view:$hash:modal")
+      .type("modal")
+      .notifyOnClose(false)
+      .title(viewTitle { it.type("plain_text").text("Full commit").emoji(true) })
+      .close(viewClose { it.type("plain_text").text("Close").emoji(true) })
```

Comment:
You probably don't need `emoji(true)` since none of those have emojis. 🙂

Reply:
oh, so that's what this was for! Just copying it. Will leave as is just in case I want to emojify it